src/signing/signing.service.spec.ts

```diff
@@ -43,7 +54,7 @@ describe('LocalSigningService', () => {
 
   describe('initialize', () => {
     it('should call polymeshApi setSigningManager method', async () => {
-      await service.initialize({});
+      await service.initialize();
```

Comment:
Apparently default args are a branch, so falling back to no args adds test branch coverage.